src/main/java/uk/gov/companieshouse/efs/api/email/model/InternalAvFailedEmailData.java

```diff
@@ -1,7 +1,9 @@
 package uk.gov.companieshouse.efs.api.email.model;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
+import java.util.function.Consumer;
 
 public class InternalAvFailedEmailData {
```

Comment:
I think this class from PR#87 got here accidentally. If so I'd revert the changes.